<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 32);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="side_menu_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	text-align: left;
	padding: 5px 1px;
    background: transparent;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	font-weight: bold;
	icon-size: 24px;
}

QPushButton:hover:!pressed {
	border-left: 3px solid green;
	background-color: gray;
}

QPushButton:checked {
	border-left: 3px solid green;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_mylist">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Library</string>
         </property>
         <property name="icon">
          <iconset resource="../desu_res.qrc">
           <normaloff>:/icons/images/library.png</normaloff>:/icons/images/library.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_main">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Main</string>
         </property>
         <property name="icon">
          <iconset resource="../desu_res.qrc">
           <normaloff>:/icons/images/main.png</normaloff>:/icons/images/main.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_shikimori">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Shikimori</string>
         </property>
         <property name="icon">
          <iconset resource="../desu_res.qrc">
           <normaloff>:/icons/images/shikimori.png</normaloff>:/icons/images/shikimori.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_history">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>History</string>
         </property>
         <property name="icon">
          <iconset resource="../desu_res.qrc">
           <normaloff>:/icons/images/history.png</normaloff>:/icons/images/history.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_settings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="icon">
          <iconset resource="../desu_res.qrc">
           <normaloff>:/icons/images/settings.png</normaloff>:/icons/images/settings.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="top_item_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	padding: 5px 1px;
	border-radius: 10px;
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="top_item">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../desu_res.qrc"/>
 </resources>
 <connections/>
</ui>
